### PR TITLE
Feat: navmap animated player compass

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?

Improvements on player visibility on the map: add pulsing circle to the player marker. 
Note: removing it from the minimap will be handled in separate branch (because low-prio for now)

...

## How to test the changes?

1. Launch the explorer
2. Open NavMap and observe pulsing circle around player marker

Please, also verify that it is not present if launching without enabled feature flag

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### <samp>🤖 Generated by Copilot at 009927f</samp>

This pull request adds and modifies some files related to the player marker on the map, which is a UI element that shows the player's position and orientation. The main changes are: adding a circle sprite to the `PlayerMarkerObject` prefab and a script to animate it, refactoring the code for creating and initializing the player marker, and changing the sorting order of the player marker layer to make it visible on top of other map elements.

### CODE REVIEW

**Comment by RominaMarchetti** [comment]
Working as expected in both cases with the FF enable (animation working) and disable (animation not working).

**Review by RominaMarchetti — APPROVED** [review]

File: unity-renderer/Assets/DCLServices/MapRendererV2/MapLayers/PlayerMarker/PlayerMarkerObject.cs [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
     internal class PlayerMarkerObject : MapRendererMarkerBase
     {
-        [field: SerializeField]
-        internal SpriteRenderer spriteRenderer { get; private set; }
+        [SerializeField] private SpriteRenderer compass;
+        [SerializeField] private SpriteRenderer circle;
+
+        private void Start()
+        {
+            if (!DataStore.i.featureFlags.flags.Get().IsFeatureEnabled("map_focus_home_or_user"))
```
Comment (davidejensen) [inline unity-renderer/Assets/DCLServices/MapRendererV2/MapLayers/PlayerMarker/PlayerMarkerObject.cs:13]: this is a bit ugly, maybe we could enavke/disable the circle from the player marker controller throught the `playermarker.cs`

**Review by davidejensen — APPROVED** [review]
approved but with minor comment

**Review by anicalbano — APPROVED** [review]
✅ Animated player compass tested and working!
✅ Not visible with FF off

https://github.com/decentraland/unity-renderer/assets/139157886/0a932b42-11f5-41d1-93c7-272a8ecb87e5

**Pull request merged**